S21/Garrett-Reed/garrett_lowercaser.py

```diff
@@ -0,0 +1,126 @@
+# 3/6/2021 - Garrett Reed - Initial Creation - Alpha#
+# Alpha version will be console-only #
+
+# Function to drive process
+def wordCaseDriver():
+    wordProcess(langEntry(), wordEntry())
+
+
+def langEntry():
+    userLang = input("Please select a language (en, en-US, en-IE, en-Latn, tr, az, ga, ga-IE, el, zh-Hans, ja, th): ")
+    langList = ["en", "en-US", "en-IE", "en-Latn", "tr", "az", "ga", "ga-IE", "el", "zh-Hans", "ja", "th"]
+    try:
+        match = langList.index(userLang)
+    except:
+        print("Error - Unknown Language")
+        quit()
+    return match
+
+
+# Accepts language codes directly instead of through the console
+def langEntry2(lang):
+    userLang = lang
+    langList = ["en", "en-US", "en-IE", "en-Latn", "tr", "az", "ga", "ga-IE", "el", "zh-Hans", "ja", "th"]
+    try:
+        match = langList.index(userLang)
+    except:
+        print("Error - Unknown Language")
+        quit()
+    return match
+
+
+def wordEntry():
+    userWord = input("Please enter the word to make lowercase: ")
+    if len(userWord) < 1:
+        print("Error - No word provided")
+        quit()
+    return userWord
+
+
+def outputLowercase(convertedWord):
+    print(convertedWord)
+    return convertedWord
+
+
+# Sends words down correct logic path
+def wordProcess(match, userWord):
+    if match == 0 or match == 1 or match == 2 or match == 3:

```

Comment:
Here, match variable has many condition comparisions and the readability is not that good. Also, if match in (0,1,2,3) this way is more Pythonic.